Std/Data/List/Lemmas.lean

```diff
@@ -206,6 +229,66 @@ theorem zipWith_get? {f : α → β → γ} :
     | nil => simp
     | cons b bs => cases i <;> simp_all
 
+@[simp]
+theorem zipWith_eq_nil_iff {f : α → β → γ} {l l'} : zipWith f l l' = [] ↔ l = [] ∨ l' = [] := by
+  cases l <;> cases l' <;> simp
+
+theorem lt_length_left_of_zipWith {f : α → β → γ} {i : Nat} {l : List α} {l' : List β}
```

Comment:
Or `lt_length_left_of_lt_zipWith_length`? Or `lt_length_of_lt_zipWith_left_length`? :-)

Reply:
Indeed that sounds better. I just changed it, but we'll have to remember to still delete it from mathlib though!